### PR TITLE
Updating to add new CFN functions created since 2017.

### PR DESCRIPTION
**Pull request by andylockran**
Unsure if this is still well maintained, but added a few more functions needed to parse CFN schemas.

### CODE REVIEW

**Comment by andyloughran** [comment]
This should address #14

File: index.js [hunk 1]
```diff
@@ -86,11 +86,40 @@ const supportedFunctions = [
...
+  'Fn::ValueOf',
+  'Fn::ValueOfAll'
+]
+
+
+let allTagTypes = []
+let allFunctions = [...supportedFunctions, ...ruleFunctions]
+
+//console.log(supportedFunctions)
+console.log(allFunctions)
```
Comment (dsagal) [inline index.js]: This seems to have been left in by accident.

**Review by dsagal — COMMENTED** [review]
Thank you! Looks good except one stray logging line.

File: index.js (hunk 1 above)
Comment (andylockran) [inline index.js]:
```suggestion
```

File: index.js [hunk 2]
```diff
@@ -86,11 +86,39 @@ const supportedFunctions = [
...
+  'Fn::RefAll',
+  'Fn::ValueOf',
+  'Fn::ValueOfAll'
+]
+
+
+let allTagTypes = []
+let allFunctions = [...supportedFunctions, ...ruleFunctions]
+
+//console.log(supportedFunctions)
```
Comment (andylockran) [inline index.js]:
```suggestion
```

**Comment by andylockran** [comment]
Resolved - thanks @dsagal

**Review by dsagal — APPROVED** [review]

**Pull request merged**